### DISCUSSION TITLE
Are submenus possible?

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2333] [Help]
Hi there,
So, my idea is to make a type of submenu and I don’t know if it is possible or not.
Here is the thing. I have a menu named Tasks. In the design section I want to make some submenus under the Tasks sections, like ‘Tasks done, tasks in progress", where in each section I only want to show the tasks that are done or the ones that are in progress. 
Is there a way to do that (they should also be seen from the client side)?

**Comment by joebudi** [comment]
You could possibly just filter the data by the status of the task?

**Reply by erakastrati** [reply]
Yeah that's probably the best way, but actually the client asked it in that way.
